app/sa_confidence/preprocess.py

```diff
@@ -0,0 +1,90 @@
+import glob
+import nltk
+import re
+import sys
+from nltk.corpus import stopwords
+from nltk.stem import WordNetLemmatizer
+
+
+# Clean sentence
+def clean_sentence(sentence):
+    # Removing all special characters
+    cleaned_sentence = re.sub("[^a-zA-Z\s0-9\-]", "", sentence)
+    # lemmatized_sentence = lemmatize_sentence(cleaned_sentence)
+    no_stopwords_sentence = remove_stopwords(cleaned_sentence)
+    # print no_stopwords_sentence
+    lemmatized_sentence = lemmatize_sentence(no_stopwords_sentence)
+    # print lemmatized_sentence
+    return lemmatized_sentence
+
+
+# Remove stopwords
+def remove_stopwords(cleaned_sentence):
+    stop_words = stopwords.words('english')
+    new_stop_words = []
+    # print len(stop_words)
+    for word in stop_words:
+        if word != 'not' and word != 'but' and word != 'if' and word != 'until' and word != 'against' and word != 'most' and word != 'no' and word != 'nor' and word != 'very' and word != 'musn\'' and word != 'needn\'' and word != 'shan\'' and word != 'shouldn\'' and word != 'wasn\'' and word != 'weren\'' and word != 'won\'' and word != 'wouldn\'':
```

Comment:
please rewrite this code in a better manner. too many conditions. create separate list for words you don't want removed.